### PR TITLE
Added exercise creation and deletion

### PR DESCRIPTION
**Pull request by craigaloewen**
Added the ability to easily create and delete exercises

### CODE REVIEW

File: SpotMe/Exercise.cs [hunk 1]
```diff
@@ -41,5 +41,81 @@ public Exercise(string inName, double[] inContractedForm, double[] inExtendedFor
...
+            classifierData.Add(newClassifier);
+            return true;
+        }
+
+        public bool DeleteClassifier(int classifierIndex)
+        {
+            if (classifierIndex < classifierData.Count)
+            {
+                classifierData.RemoveAt(classifierIndex);
+                reIndexClassifiersStartingAt(classifierIndex);
```
Comment (jhonyguan) [inline SpotMe/Exercise.cs:58]: You may want to delete the classifier and training data files too. we don't want too many unusable files

File: SpotMe/Classifier.cs [hunk 2]
```diff
@@ -50,6 +60,26 @@ public Classifier(int inId, string inName, string inMessage, SkeletonForm inForm
...
+            TrainingData newData = new SpotMe.TrainingData(this, inputData);
+            formTrainingData.Add(newData);
+            return true;
+        }
+
+        public bool DeleteTrainingData(int trainingDataIndex)
+        {
+            if ( trainingDataIndex < formTrainingData.Count )
+            {
+                formTrainingData.RemoveAt(trainingDataIndex);
```
Comment (jhonyguan) [inline SpotMe/Classifier.cs:74]: Same here, you may want to delete trainingdata files if they exist. The deleteFunctions have file check I think.

**Review by jhonyguan — COMMENTED** [review]
Consider deleting files when deleting things from the Model

File: SpotMe/Exercise.cs (hunk 1 above)
Comment (craigaloewen) [reply SpotMe/Exercise.cs:58]: If we have them all consolidated into one file:
exerciseData/
    bicepCurl.txt
   militaryPress.txt

then you just need to write a saveExercise function and delete the file if it already exists

**Pull request closed without merging**